static/content.less

```diff
@@ -141,3 +141,7 @@ blockquote{
 .body{
   margin: @gutter*2;
 }
+ul.community-list {
```

Comment:
I’m hoping we’ll reuse this same pattern (icons next to list items), so let’s call it something like `icon-list` (or something better… naming is hard!).